examples/animatediff/args_train.py

```diff
@@ -17,6 +17,15 @@ def _check_cfgs_in_parser(cfgs: dict, parser: argparse.ArgumentParser):
             raise KeyError(f"{k} does not exist in ArgumentParser!")
 
 
+def _parse_bool_str(b: str):
+    """Allow input args to be either str2bool or str (e.g. a filepath)."""
+    if b.lower() not in ["false", "true"]:
+        return b
+    if b.lower() in ["false"]:
+        return False
+    return True
```

Comment:
How about modifying `mindone.utils.config.str2bool` by adding to it a `return_str` parameter? Something like:
```python
def str2bool(b: str, return_str: bool = False):
    if b.lower() not in ["false", "true"]:
        if return_str:
            return b
        else:
            raise Exception("Invalid Bool Value")
...
```